src/common/gui/ThemeController.ts

```diff
@@ -24,7 +24,12 @@ export class ThemeController {
 	readonly observableThemeId: Stream<ThemeId>
 	readonly initialized: Promise<any>
 
-	constructor(themeSingleton: {}, private readonly themeFacade: ThemeFacade, private readonly htmlSanitizer: () => Promise<HtmlSanitizer>) {
+	constructor(
+		themeSingleton: {},
+		private readonly themeFacade: ThemeFacade,
+		private readonly htmlSanitizer: () => Promise<HtmlSanitizer>,
+		private readonly deviceConfig: DeviceConfig,
```

Comment:
We overwrite existing themes, all this was to deal with the blue theme haha
Since it doesn't exist in the theme object anymore, it's treated as a white label theme. But it's totally possible to check separately if the blue theme exists

Reply:
I would probably just manually check for blue so that we don't have people stuck with it and got rid of migration flag